conf/minifi.properties

```diff
@@ -37,10 +37,37 @@ nifi.database.content.repository.directory.default=${MINIFI_HOME}/content_reposi
 #nifi.rest.api.user.name=admin
 #nifi.rest.api.password=password
 
+## Enabling C2 Uncomment each of the following options
+## define those with missing options
+#nifi.c2.enable=true
+## define protocol parameters
+#c2.agent.protocol.class=CoapProtocol
```

Comment:
We need to update the language surrounding this PR's changes in C2.md as well
https://github.com/apache/nifi-minifi-cpp/blob/master/C2.md#protocols

Reply:
Sorry had a dirty repo with other changes and didn't include this . Thanks!